doc/plan/features.md

```diff
@@ -32,4 +32,6 @@ just show total.
 ## Mobile Device Support
 
 Currently using `MediaQuery.of()` to get device width.
-Evaluation [responsive_framework](https://pub.dev/packages/responsive_framework) for potential use.
\ No newline at end of file
+Evaluation [responsive_framework](https://pub.dev/packages/responsive_framework) for potential use.
+
+Change status screen to support Android. --Jesse
```

Comment:
When are you going to do this by?  Do you have the Android emulator running on Windows?  You need to test it.

Reply:
I will have this done by EOD tomorrow (Thurs, Oct 29). I have the Android emulator running on Windows.

Reply:
OK.   We likely need need to get the responsive (auto-resizing) features at some point because we intend to use Flutter Web for Oppkey software projects early 2021.  This is just for practice.